lib/Screens/Login.dart

```diff
@@ -0,0 +1,162 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+class SignIn extends StatelessWidget {
```

Comment:
name of file should be the same as file name, so rename Login.dart to sing_in.dart or rename SignIn class to Login class. Also please keep packages and files name in lowercase_with_underscores convention. So instead Screens/Login.dart make packages and files like screens/login.dart